README.md

```diff
@@ -92,16 +113,43 @@ $ docker run --env "HC_GITHUB_TOKEN=<GITHUB_TOKEN>" hipheck:3.1.0 [<HIPCHECK_ARG
 
 ### Direct Usage
 
-You can run Hipcheck with the `hc` binary.
+To run Hipcheck, make sure you export `HC_GITHUB_TOKEN` with a valid token for
```

Comment:
Strictly speaking you don't _have_ to set `HC_GITHUB_TOKEN` if you are willing to use the appropriate flag every time you run Hipcheck. Maybe change the language to say that we recommend doing this to avoid having to specify the token when running Hipcheck.

Reply:
What portion of the code implements that cmdline arg? When I look at `analysis/session/mod.rs::resolve_token()` it looks like it only checks `HC_GITHUB_TOKEN`

Reply:
Oh never mind.  We had that as a feature at one point but we didn't implement it.